packages/cli/src/handlers/dbt/apiClient.ts

```diff
@@ -48,16 +53,20 @@ export const lightdashApi = async <T extends ApiResponse['results']>({
         })
         .then((r) => r.json())
         .then((d: ApiResponse | ApiError) => {
+            if (verbose)
+                console.error(`> HTTP request returned status: ${d.status}`);
+
             switch (d.status) {
                 case 'ok':
                     return d.results as T;
                 case 'error':
-                    throw d;
+                    throw new Error(`${d}`);
```

Comment:
Not sure why I had to update this error  now, I think I updated my eslint dependency

![Screenshot from 2022-09-22 09-45-48](https://user-images.githubusercontent.com/1983672/191688687-0fbc6b47-ab48-470b-9076-fea6ec987e38.png)